AndroidAnnotations/androidannotations-core/androidannotations/src/test/java/org/androidannotations/generation/GenerateJavaDocTest.java

```diff
@@ -67,4 +67,17 @@ public void generateJavaDocForServiceAction() throws IOException {
 		assertGeneratedClassMatches(generatedFile, ".*\\*  @param param this is a param");
 		assertGeneratedClassMatches(generatedFile, ".*\\* @return");
 	}
+
+	@Test
+	public void generateJavaDocForSharedPref() throws IOException {
+		CompileResult result = compileFiles(SharedPrefWithJavaDoc.class);
+		File generatedFile = toGeneratedFile(SharedPrefWithJavaDoc.class);
+
+		assertCompilationSuccessful(result);
+
+		assertGeneratedClassMatches(generatedFile, ".*\\* <p><b>Defaults to</b>: 42</p>");
```

Comment:
It would be more correct to use `assertGeneratedClassContains`, since we have to check that both the normal and the editor method has the javadoc.